crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,location/url[@displayName='Download URL']
+fileSize,physicalDescription/extent
+installUrl,location/url[@displayName='Install URL']
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,description
+softwareHelp,relatedItem
+softwareRequirements,
+softwareVersion,originInfo/edition
+storageRequirements,
+supportingData,relatedItem[@type='references']
+author,name/role/roleTerm[@type='code']/aut
+citation,relatedItem[@otherType='references']
+contributor,name/role/roleTerm[@type='code']/ctb
+copyrightHolder,accessCondition/rights.holder/name
+copyrightYear,accessCondition/creation/year.creation
+dateCreated,originInfo/dateCreated
+dateModified,originInfo/dateModified
+datePublished,originInfo/dateIssued
+editor,name/role/roleTerm[@type='code']/edt
+encoding,
+fileFormat,typeOfResource
+funder,name/role/roleTerm[@type='code']/fnd
+keywords,subject
+license,accessCondition[@type='use and reproduction']
+producer,name/role/roleTerm[@type='code']/pro
+provider,name/role/roleTerm[@type='code']/prv
+publisher,originInfo/publisher
+sponsor,name/role/roleTerm[@type='code'
+version,originInfo/edition
+isAccessibleForFree,accessCondition[@type='accessible for free']/yes

```

Comment:
Do you have a reference for this one? I don't see it in the examples here: https://www.loc.gov/standards/mods/userguide/accesscondition.html

Reply:
```suggestion
isAccessibleForFree,
```
After revisiting this, I think it may be better to not map the isAccessibleForFree attribute without a controlled list for access condition type values.